apps/backend/src/app/api/v1/internal/metrics/route.tsx

```diff
@@ -117,11 +118,44 @@ function simplifyUsers(users: (ProjectUser & { contactChannels: ContactChannel[]
     id: user.projectUserId,
     display_name: user.displayName,
     email: user.contactChannels.find(x => x.isPrimary)?.value ?? '-',
-    created_at: user.createdAt.getTime(),
-    updated_at: user.updatedAt.getTime(),
+    created_at_millis: user.createdAt.getTime(),
+    updated_at_millis: user.updatedAt.getTime(),
   }));
 }
 
+async function loadRecentlyActiveUsers(projectId: string):
+  Promise<(ProjectUser & { contactChannels: ContactChannel[] })[]> {
+
+  // use the Events table to get the most recent activity
+  const events = await prismaClient.event.findMany({
+    take: 10,
```

Comment:
What will this do when the 10 most recent events are all from the same user?

Reply:
We could probably query for distinct, but Prisma doesn't support that either so it will probably require raw queries?